src/block-ciphers.rst

```diff
@@ -116,48 +116,48 @@ arrows, as you can see in :numref:`fig-BlockCipherEncryptionDifferentKey`.
 
    An encryption permutation produced by the block cipher under some other key.
 
-In this illustration, you'll even notice that there are two permutations
-of length 1: an element that maps to itself. This is again something to
-be expected when selecting random permutations.
-
-Knowing a bunch of (input, output) pairs for a given key shouldn't give
-you any information about any other (input, output) pairs under that
-key [#]_. As long as we're talking about a hypothetical perfect block
-cipher, there's no easier way to decrypt a block other than to
-“brute-force” the key: i.e. just try every single one of them until you
+In this illustration, you will notice that there are two permutations
+of length 1. An element maps to itself. This is expected when selecting 
+random permutations.
+
+Knowing many (input, output) pairs for a given key should not give
+you any information about other (input, output) pairs under that
+key [#]_. In a hypothetically perfect, block
+cipher, there is no simple way to decrypt a block other than to
+“brute-force” the key. For example, just try every key until you
 find the right one.
 
 .. [#]
-   The attentive reader may have noticed that this breaks in the
-   extremes: if you know all but one of the pairs, then you know the
+   The attentive reader may notice that this breaks into the
```

Comment:
This change doesn't seem correct? I'm saying this idea doesn't work in the limit. I'm not suggesting that the idea _is_ itself extreme.

Reply:
@nktrejo2020 this is the change I was thinking of, btw

Reply:
@nktrejo2020 
It breaks in an extreme case

Reply:
FWIW a common math way to say this is "it breaks in the limit" instead of "the extremes". We could also just drop this block back to the original.